test/suites/rt-upgrade/test-upgrade-chain.ts

```diff
@@ -0,0 +1,63 @@
+import { describeSuite, beforeAll, expect } from "@moonwall/cli";
+import { alith } from "@moonwall/util";
+import { ApiPromise } from "@polkadot/api";
+import { parseEther } from "ethers";
+import "@moonbeam-network/api-augment";
+
+describeSuite({
+  id: "CHU",
+  title: "Chopsticks Upgrade",
+  foundationMethods: "chopsticks",
+  testCases: ({ it, context }) => {
+    let api: ApiPromise;
+    const DUMMY_ACCOUNT = "0x11d88f59425cbc1867883fcf93614bf70e87E854";
+
+    beforeAll(() => {
+      api = context.polkadotJs();
+    });
+
+    it({
+      id: "T1",
+      title: "Calling chain constants data",
+      test: async () => {
+        const specVersion = api.consts.system.version.specVersion.toNumber();
+        expect(specVersion).to.be.greaterThan(0);
+      },
+    });
+
+    it({
+      id: "T2",
+      title: "Do an upgrade test",
+      timeout: 30000,
+      test: async function () {
+        const rtBefore = api.consts.system.version.specVersion.toNumber();
+        await context.upgradeRuntime(context);
+        const rtafter = api.consts.system.version.specVersion.toNumber();
+        expect(rtBefore).toBeLessThan(rtafter);
```

Comment:
This will fail for PR after the release is deployed but before the runtime bump version merge, as both the PR runtime version and the network version will be the same. I think it is acceptable (as it will be rare) but should probably be mentioned in the error message

Reply:
This is only getting run when `Publish Runtime Draft` workflow is run`, not on each pr, so this scenario is avoided entirely. 

I figured we would only want to run this test just before we are ready to cut a new RT version as tip of master will not always be prod-ready

Reply:
Ok but if it is blocking the release, that might be an issue, as we sometime have to rush it.

Reply:
Maybe having it run when merging to master a PR